drape_frontend/renderer3d.cpp

```diff
@@ -0,0 +1,116 @@
+#include "renderer3d.hpp"
+
+#include "drape/data_buffer.hpp"
+#include "drape/glconstants.hpp"
+#include "drape/glfunctions.hpp"
+#include "drape/glstate.hpp"
+#include "drape/shader_def.hpp"
+#include "drape/uniform_values_storage.hpp"
+
+#include "math.h"
```

Comment:
Is it a local math.h? If yes, it's better to rename it to avoid clash with <math.h>. If not, then please include std/math.hpp.